src/styles/theme/themes/dark.ts

```diff
@@ -87,6 +88,8 @@ const darkTheme = {
     starDefaultBG: 'rgb(254, 228, 94)',
     mapAttributionText: colors.black,
     white: colors.white,
+    progressBarBackground: colors.productDark400,
```

Comment:
Is this the same as our border color? If so, maybe we can grab borders instead since that's going to work better cross-theme?